analyses/analysis_example/scripts/analysis_example.py

```diff
@@ -0,0 +1,41 @@
+#Mandatory: List of processes
+processList = {
+    'p8_noBES_ee_H_Hbb_ecm125':{'fraction':0.01, 'chunks':1, 'output':'test_out'}
+}
+
+#Mandatory: Production tag when running over EDM4Hep centrally produced events, this points to the yaml files for getting sample statistics
+prodTag     = "FCCee/spring2021/IDEA/"
+
+#Optional: output directory, default is local running directory
+outputDir   = "."
+
+#Optional
+nCPUS       = 8
+runBatch    = False
+#batchQueue = "longlunch"
+#compGroup = "group_u_FCC.local_gen"
+
+#Optional test file
+testFile ="root://eospublic.cern.ch//eos/experiment/fcc/ee/generation/DelphesEvents/spring2021/IDEA/p8_ee_ZH_ecm240/events_101027117.root"
+
+#Mandatory: RDFanalysis class where the use defines the operations on the TTree
+class RDFanalysis():
+    import ROOT
+    print("----> Load cxx analyzers from analysis_example... ",)
+    ROOT.gSystem.Load("libFCCAnalysis_analysis_example")
+    _dum = ROOT.analysis_example.dictionary
+    ROOT.analysis_example.dummy_analysis()
```

Comment:
Not sure why do you need to do both here?

Reply:
That was only for the sake of testing the call to a compiled library's static function. Can safely be removed!